build/docs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
```

Comment:
moved `docs.sh` to `release` dir but still need to implement to match convention of the other scripts